src/hooks/useUpdateSetting.jsx

```diff
@@ -16,6 +16,10 @@ const useUpdateSetting = (type) => {
       [type]: !settings[type],
     };
 
+    if (type === "isAdDetect" && !updatedSettings.isAdDetect) {
```

Comment:
```suggestion
    if (type === SETTING_TYPES.AD_DETECT && !updatedSettings.isAdDetect) {
```
"isAdDetect" 값을 직접 문자열로 사용하는 대신,
이미 정의해두신 상수를 활용하면 유지보수나 오타 방지 측면에서 더 안전할 것 같아요 :)

Reply:
좋은 리뷰 감사드립니다! 해당 사항 반영하겠습니다~